src/html-imports.js

```diff
@@ -347,6 +347,9 @@
         n.setAttribute(importDependencyAttr, '');
         // Generate source map hints for inline scripts.
         if (n.localName === 'script' && !n.src && n.textContent) {
+          if(n.type === 'module') {
+            throw new Error('Inline module scripts are not supported in HTML Imports.');
```

Comment:
Can we add another test to assert on this "new" behavior?

Reply:
I tried, but I was unable to catch the error within mocha. Do you have any ideas?

Reply:
Hm, I think you can dynamically create the link, append it to the body and wrap that in a try-catch?

Reply:
I tried that before, but the error does not end up in the try catch because the import is executed later (due to the mutation observer). I got it working now though using window.error!